Gemfile

```diff
@@ -48,6 +48,8 @@ gem 'jquery-rails', '~> 4.3', '>= 4.3.5'
 
 gem 'chartkick', '~> 3.3', '>= 3.3.1'
 
+gem 'clockwork', '~> 1.2'
```

Comment:
I'm not really a fan of clockwork, I think these could be better alternatives: 
https://github.com/ondrejbartas/sidekiq-cron
https://github.com/moove-it/sidekiq-scheduler

wdyt?

Reply:
Great! I don't need a new worker on Heroku. Thanks for the approach